assets/js/translator.js

```diff
@@ -133,8 +133,6 @@ if(modeEnabled('translation')) {
             synchronizeTextareaHeights();
         });
 
-        $(window).resize(synchronizeTextareaHeights);
```

Comment:
Using git blame, it shows that you have made the latest modification to that line with commit message as in this link. How do I infer from this?